gooddata-java-model/src/test/resources/project/projects.json

```diff
@@ -44,6 +44,7 @@
                         "eventstores": "/gdc/projects/defaultEmptyProject/dataload/eventstore/stores",
                         "execute": "/gdc/projects/defaultEmptyProject/execute",
                         "clearCaches": "/gdc/projects/defaultEmptyProject/clearCaches",
+                        "projectFeatureFlags": "/gdc/projects/defaultEmptyProject/projectFeatureFlags",
```

Comment:
Shouldn't we just introduce something like `"alreadyDeprecatedLink": "...",` to properly illustrate that we are able to accept deprecated links? 🤷

Reply:
Good idea, added "unknownLink" ;)